package_linter.py

```diff
@@ -1413,6 +1413,18 @@ def progression(self):
                 "during the installation. (and ideally in scripts remove, upgrade and restore too)"
             )
 
+    @test(only=["backup"])
+    def progression_in_backup(self):
+        if self.contains("ynh_script_progression"):
+            yield Info(
+                "We recommend to *not* use 'ynh_script_progression' in backup "
```

Comment:
This is very specific, but FYI, gitlab takes time to create a backup: https://github.com/YunoHost-Apps/gitlab_ynh/blob/10cc98b33461b318273c6645fa48b3364f4ac45d/scripts/backup#L51

Reply:
Zblerg yeah I guess there are edge cases, hence reporting this as an "info" and it's mainly cosmetics

I think I would still use a regular info message because 
- the point of progress bar is to have a notion of progress relative to the other things being performed before/after ... but if one operation takes 99% of the time, there isn't so much sense in this
- the backup script is called during the upgrade, which itself has a progression bar ... and that makes things funky because there's a progression bar "inside" the progression bar

So would instead just have a message saying "this will take a while!"

But again, really not the biggest deal ever (but I understand it's annoying having messages in the linter that you can't really fix even though they ain't warning :s)